app/helpers/link_helper.rb

```diff
@@ -4,4 +4,8 @@ def active_link_class(link_path)
 
     "govuk-header__navigation-item"
   end
+
+  def open_in_new_tab_link_to(text, href, **kwargs)
+    govuk_link_to("#{text} (opens in new tab)", href, target: "_blank", rel: "noreferrer noopener", **kwargs)
```

Comment:
Edge case but what happens if kwargs includes some other `rel` attributes? Do they overwrite or merge with the noferrer/noopener?